servers.json

```diff
@@ -13,6 +13,16 @@
             "edenmine.eu"
         ]
     },
+    {
+        "id": "baldymc",
+        "name": "BaldyMC",
+        "addresses": [
+            "baldymc.com",
+            "baldymc.xyz",
+            "play.baldymc.com",
```

Comment:
Please take a look at:
[the ip-address section in the README](https://github.com/LunarClient/ServerMappings/blob/master/README.md#ip-addresses)

You do not need to add sub-domains to the addresses array.

Reply:
forgot sorrry